apps/api-extractor-model/src/model/ApiEntryPoint.ts

```diff
@@ -51,4 +53,23 @@ export class ApiEntryPoint extends ApiItemContainerMixin(ApiNameMixin(ApiItem))
     // No prefix needed, because ApiEntryPoint is the only possible member of an ApiPackage
     return this.name;
   }
+
+  /** @beta @override */
+  public buildCanonicalReference(): DeclarationReference {
+    let modulePath: string = '';
+
+    if (this.parent instanceof ApiPackage) {
+      modulePath = this.parent.name;
+    }
+
+    if (this.name) {
+      modulePath += this.name;
+    }
+
+    if (modulePath) {
+      return new DeclarationReference(new ModuleSource(modulePath));
+    }
+
+    return DeclarationReference.empty();
```

Comment:
This seems like it should be an error?

Reply:
A `ApiEntryPoint` can exist in a parentless state before it has been added to its `ApiPackage` parent.  Property getters aren't supposed to throw exceptions, so in this detatched state it needs to return some kind of value.